packages/autometrics-lib/src/wrappers.ts

```diff
@@ -64,19 +64,18 @@ type AnyFunction<T extends FunctionSig> = (
 // rome-ignore lint/suspicious/noEmptyInterface: Converting this to a type breaks the language server plugin
 interface AutometricsWrapper<T extends AnyFunction<T>> extends AnyFunction<T> {}
 
-export type AutometricsOptions =
-  | {
-      /**
-       * Name of your function
-       */
-      functionName: string;
-      /**
-       * Name of the module (usually filename)
-       */
-      moduleName?: string;
-      objective?: Objective;
-    }
-  | { objective: Objective };
+export type AutometricsOptions = {
+  /**
+   * Name of your function
+   */
+  functionName?: string;
+  /**
+   * Name of the module (usually filename)
+   */
+  moduleName?: string;
+  objective?: Objective;
+  trackConcurrency?: boolean;
```

Comment:
can we add a jsdoc comment for the `trackConcurrency` option? Especially highlighting what is the default param value and what it expects.

(also for the `objective` 🥺)

Reply:
Good call, updated!